taxonomy/definitions.md

```diff
@@ -0,0 +1,21 @@
+# Glossary of Terms: Understanding Lifecycle Phases in Technology
+
+This document aims to clearly define key terms related to the lifecycle of software and hardware products, providing a reference for understanding various phases from sales or creation to support termination. 
+
+**Note**: With open source software or hardware, there is typically no "end-of-sales".
+
+## 1. Software End-of-Life (SEoL) or Software End-of-Sales (SEoS)
+
+**Definition:** Software End-of-Life refers to the stage in a software product's lifecycle where it is no longer sold. 
+
+## 2. Hardware End-of-Life (HEoL) or Hardware End-of-Sales (HEoS)
+
+**Definition:** Hardware End-of-Life signifies the phase where a piece of hardware (like a server, router, or computer component) is no longer sold by its manufacturer.
+
+## 3. End-of-Software Maintenance or Security Vulnerability Support
```

Comment:
I think those are two distinct things:
1. End of Software Maintenance: No bugfixes, no new feature
2. End of Security vulnerability Support: no more security fixes

This also reflects what is currently done by many open source projects (not backporting new features, but keeping security up-to-date for a while) and commercial products (e.g. Windows 10)